src/setOfPeople.js

```diff
@@ -38,7 +38,15 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  let roomWithPeople = new Set();
```

Comment:
[prefer-const](https://github.com/mate-academy/js_array-of-people/pull/7#discussion_r275119762)